recipes/datashuttle/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "datashuttle" %}
+{% set version = "0.1.0rc2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 982b14ac8f3976a573b1e2329db817b3bbd95f7ce0495ba16a648d368853d287
+
+build:
+  noarch: python
+  entry_points:
+    - "datashuttle = datashuttle.command_line_interface:main"
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8.0
+    - pip
+  run:
+    - python >=3.8.0
+    - paramiko
+    - pyyaml
+    - requests
+    - rich
+    - fancylog
+    - simplejson
+    - rclone
+
+test:
+  imports:
+    - datashuttle
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://datashuttle.neuroinformatics.dev/
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: "A project folder manager for systems neuroscience."
+
+extra:
+  recipe-maintainers:
+    - JoeZiminski
+    - adamltyson
+    - niksirbi
```

Comment:
I think we just need @adamltyson and @niksirbi to give a thumbs up to be listed as maintainers here.

Reply:
Fine by me!

Reply:
Fine by me too!